src/resources/order/models/index.model.js

```diff
@@ -1,7 +1,27 @@
 import Order from './order.model';
 
-async function createOrder(order) {
-  return Order.query().insert(order).returning('*');
+async function createOrder(order, products) {
+  try {
+    return await Order.transaction(async (trx) => {
+      const transformedProducts = products.map((prod) => {
```

Comment:
I'm guessing this is what lets you insert more than one product at a time. Nice, is this from objection js?